pontoon/contributors/views.py

```diff
@@ -183,16 +183,8 @@ def update_contribution_timeline(request):
 @login_required(redirect_field_name="", login_url="/403")
 @require_POST
 @transaction.atomic
-def edit_user_profile_fields(request, username):
-    user = get_object_or_404(User, username=username)
-    if user != request.user:
-        return JsonResponse(
-            {
-                "status": False,
-                "message": "Forbidden: You don't have permission to edit this user",
-            },
-            status=403,
-        )
+def edit_user_profile_fields(request):
+    user = request.user
```

Comment:
There are a few places below that should refer to `user` rather than `request.user`.